statsmodels/base/distributed_estimation.py

```diff
@@ -0,0 +1,410 @@
+from statsmodels.base.elastic_net import fit_elasticnet
+from statsmodels.base.model import Results
+from statsmodels.tools.decorators import cache_readonly
+from statsmodels.regression.linear_model import OLS
+#from statsmodels.genmod.generalized_linear_model import GLM
+#from statsmodels.genmod.families import Binomial
+import statsmodels.base.wrapper as wrap
+import numpy as np
+
+"""
+distributed regularized estimation.
+
+Routines for fitting regression models using a distributed
+approach outlined in 
+
+Jason D. Lee, Qiang Liu, Yuekai Sun and Jonathan E. Taylor.
+"Communication-Efficient Sparse Regression: A One-Shot Approach." 
+arXiv:1503.04337. 2015.
+"""
+
+
+def _gen_grad(mod, beta_hat, n, alpha, L1_wt, score_kwds):
```

Comment:
We try to avoid using greek letters since they don't have a lot of meaning in themselves.  It's OK for an internal function but try to minimize.  For example, we usually use "params" instead of "beta".  We are currently using "alpha" as a regularization parameter for legacy reasons but may likely rename it in the future.